docs/odk-workflows/RepositoryFileStructure.md

```diff
@@ -34,6 +34,7 @@ These are the components in MONDO-INGEST
 | gard.owl | https://github.com/monarch-initiative/gard/releases/latest/download/gard.owl |
 | icd10cm.owl | https://data.bioontology.org/ontologies/ICD10CM/submissions/23/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb |
 | icd10who.owl | https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl |
+| icd11.owl | https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz |
```

Comment:
## `icd11.owl` --> `icd11.ofn`?
This seems less ambiguous to me. Or maybe `icd11.owl.ofn`. Normally I when I see `.owl` it's RDF/XML. Though I suppose for those `.owl.rdf.xml` or `.owl.xml` might be better, but we meh.

Reply:
You are of course right that ofn would be less ambiguos. But we use owl all the time to mean anything from rdfxml to functional syntax. ROBOT does not care about the file ending, so I would be inclined to continue to use functional syntax plus .owl, which is what the other components are doing as well.